openshift/deploy/job/schema-revert.yml

```diff
@@ -64,7 +64,7 @@ objects:
           - name: PGPORT
             value: "5432"
           - name: PGHOST
-            value: ${PREFIX}postgres
+            value: cas-postgres-master
```

Comment:
`${PREFIX}postgres-master`, no?

Reply:
${PREFIX} is 'cas-ggircs' in this repo...

Reply:
gotcha